VoxelSniperCore/src/main/java/com/github/kevindagame/voxelsniper/fileHandler/VoxelSniperConfiguration.java

```diff
@@ -61,13 +66,22 @@ public void setMessageOnLoginEnabled(boolean enabled) {
         configuration.set(CONFIG_IDENTIFIER_MESSAGE_ON_LOGIN_ENABLED, enabled);
     }
 
+    /**
+     * Returns the default brush.
+     *
+     * @return the default brush.
+     */
+    public String getDefaultBrush() {
+        return configuration.getString(CONFIG_IDENTIFIER_DEFAULT_BRUSH);
```

Comment:
Could we add a default value here, so that this doesn't break when the setting isn't present in the config file (e.g. old installs)?

Reply:
Strings don't allow a default value on the yaml class. Default is already empty string

Reply:
Than I think we should add an option to have default values. Could you do that, or should I?

Reply:
I don't see the point. An empty value is already being handled as an empty string

Reply:
done ;)